src/components/PeopleTable.tsx

```diff
@@ -1,52 +1,101 @@
-export const PeopleTable = () => {
+import React from 'react';
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { SortBy } from '../types/SortBy';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  peopleList: Person[],
+  selectedPerson: string | null,
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  peopleList,
+  selectedPerson,
+}) => {
+  const isSelected = (person: Person) => person.slug === selectedPerson;
+  const [searchParams] = useSearchParams();
+  const location = useLocation();
+  const sort = searchParams.get('sort') as SortBy || null;
+  const order = searchParams.get('order');
+
+  const getSlug = (name: string | null) => {
+    const findPerson = peopleList.find(person => person.name === name);
+
+    return findPerson?.slug;
+  };
+
+  const handleSortPeople = (sortBy: SortBy) => {
+    return {
+      sort: sort === sortBy && order ? null : sortBy,
+      order: sort === sortBy && !order ? 'desc' : null,
+    };
+  };
+
+  const handleSortClass = (sortBy: SortBy) => {
+    return classNames(
+      'fas',
+      { 'fa-sort': sortBy !== sort },
+      { 'fa-sort-up': sortBy === sort && !order },
+      { 'fa-sort-down': sortBy === sort && order },
+    );
+  };
+
   return (
     <table
       data-cy="peopleTable"
-      className="table is-striped is-hoverable is-narrow is-fullwidth"
+      className="
```

Comment:
Consider joining the class names into a single line to improve code readability.